mpu6050/python/mpu6050Python/.vscode/settings.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Please add this file without configurations, like [this](https://github.com/toradex/torizon-samples/commit/27c0e8b31e7d453f8e2a689936f577f634b31d5d#diff-e5bd81f315f256b141250aeffcabbbc22850c602822ad19d12ad6492fa68d94b)